jsdoc-config.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
No, file was needed to test compliance and accidentally got committed.

> On May 23, 2016, at 09:46, Yoshiki Vázquez Baeza <notifications@github.com> wrote:
> 
> In jsdoc-config.json <https://github.com/biocore/emperor/pull/485#discussion_r64249417>:
> 
> > @@ -0,0 +1,19 @@
> > +{
> Did the commits get scrambled?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/biocore/emperor/pull/485/files/c065e576ca428d9626fb0e5cd9d48a745c5e4124#r64249417>